pkg/infra/pulumi_aws/iac/eks.ts

```diff
@@ -545,14 +553,40 @@ export class Eks {
         if (chart == undefined) {
             throw new Error('Cannot create ServiceExport without Cloud Map Controller installed')
         }
+
+        let sleep
+        if (this.options.initializePluginsOnFargate) {
+            sleep = new local.Command(
```

Comment:
minor: it looks like most of the if-else blocks are identical, with just the sleep quantity different. If so, you could simplify this a bit imo to make it clear exactly what's different between the two branches:

```
let sleepAmount = this.options.initializePluginsOnFargate ? 60 : 10;
let sleep = new local.Command(...
    create: `sleep ${sleepAmount}`,
...
```

Reply:
ah yeah ill change this, added things in phases so just a bit of brain mush in here like this